<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crear</class>
 <widget class="QDialog" name="crear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QLineEdit" name="inputCodigo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border: 1px solid #3498DB ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>codigo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputNombre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border: 1px solid #3498DB ;</string>
   </property>
   <property name="placeholderText">
    <string>nombre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonActualizar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#2ECC71;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>ACTUALIZAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btncancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#E74C3C;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>OFERTA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green;</string>
   </property>
   <property name="text">
    <string>ENVIO GRATIS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputCategoria">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border: 1px solid #3498DB ;</string>
   </property>
   <property name="placeholderText">
    <string>categoria</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPrecio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border: 1px solid #3498DB ;</string>
   </property>
   <property name="placeholderText">
    <string>precio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputUnidades">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border: 1px solid #3498DB ;</string>
   </property>
   <property name="placeholderText">
    <string>unidades</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputCodigoProducto">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border: 1px solid #3498DB ;</string>
   </property>
   <property name="placeholderText">
    <string>codigo del producto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBuscar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(26, 75, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>BUSCAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
